Sources/BraveWallet/Settings/DappsSettings.swift

```diff
@@ -323,9 +309,6 @@ private struct SiteConnectionDetailView: View {
           Text(Strings.Wallet.manageSiteConnectionsConfirmAlertRemove),
           action: {
             siteConnectionStore.removeAllPermissions(from: [siteConnection])
-            if #unavailable(iOS 15) { // iOS 15 will dismiss itself
-              presentationMode.dismiss()
-            }
```

Comment:
These changes are also iffy, mostly because the comment indicated iOS 15 would dismiss itself but then its only executing on 14 😅 

Let me know if we need to keep the dismiss call ehre